docker/build.sh

```diff
@@ -1,4 +1,22 @@
-cd base && ./build.sh && cd ..
+BASE_IMAGE=$1
+
+# find L4T_VERSION
+source ../scripts/l4t-version.sh
+
+if [ -z $BASE_IMAGE ]; then
+	if [ $L4T_VERSION = "32.4.4" ]; then
```

Comment:
Is there a reason to drop 32.4.3 which was previously handled in docker/base/build.sh?

Reply:
Also, can we move this logic to configure.sh as an environment variable?  This is currently where all other docker build configuration is done.